src/containers/my-cooperations/cooperation-details/CooperationDetails.tsx

```diff
@@ -74,12 +82,21 @@ const CooperationDetails = () => {
 
   const cooperationContent = activeTab && tabsData[activeTab]?.content
 
-  const pageContent =
-    isActivityCreated && activeTab === CooperationTabsEnum.Activities ? (
-      <CooperationActivities cooperationId={id} />
-    ) : (
-      cooperationContent
-    )
+  const pageContent = editMode ? (
+    (editMode === true &&
+      CooperationTabsEnum.Details === activeTab &&
+      cooperationContent) ||
+    (editMode === true && activeTab === CooperationTabsEnum.Activities && (
+      <CooperationActivitiesView
+        sections={response.sections}
+        setEditMode={setEditMode}
+      />
+    ))
+  ) : isActivityCreated ? (
+    <CooperationActivities cooperationId={id} />
+  ) : (
+    cooperationContent
+  )
```

Comment:
I suggest using render function instead because this code is not readable. I suggest refactoring like following:

```
const renderPageContent = () => {
    if (editMode) {
         return ...
    }
    
    if (isActivityCreated) {
         return ...
    }
    
    return ...
}
```

Reply:
fixed